browser-extension/mv2/src/client/js/scriptRuleHandler.js

```diff
@@ -128,24 +128,105 @@ RQ.ScriptRuleHandler.includeJSScriptsInOrder = function (scripts, callback, inde
 };
 
 RQ.ScriptRuleHandler.includeJS = function (script, callback) {
+  if (!script.value) throw new Error("Script value is empty");
   if (script.type === RQ.SCRIPT_TYPES.URL) {
+    if (script.wrapperElement) {
+      const scriptBlock = RQ.ScriptRuleHandler.parseSpecificHTMLStringBlocks(script.wrapperElement, "script")?.[0];
+      if (scriptBlock) {
```

Comment:
Too many nested ifs. Can we convert script.type check into switch-case? Can we create another function which takes the required args and return the scriptBlock to be embedded?